main/src/ca/uwaterloo/flix/language/fmt/FormatType.scala

```diff
@@ -17,14 +17,40 @@ package ca.uwaterloo.flix.language.fmt
 
 import ca.uwaterloo.flix.api.Flix
 import ca.uwaterloo.flix.language.ast.Ast.VarText
-import ca.uwaterloo.flix.language.ast.{Kind, SourceLocation, Symbol, Type}
+import ca.uwaterloo.flix.language.ast.{Kind, RigidityEnv, SourceLocation, Symbol, Type}
+import ca.uwaterloo.flix.language.phase.unification.{Substitution, TypeMinimization}
 
 object FormatType {
   /**
     * Transforms the given well-kinded type into a string.
     */
-  def formatType(tpe: Type)(implicit flix: Flix): String = {
-    formatTypeWithOptions(tpe, flix.getFormatOptions)
+  def formatType(tpe: Type, renv: RigidityEnv = RigidityEnv.empty)(implicit flix: Flix): String = {
+    val minimized = TypeMinimization.minimizeType(tpe)
```

Comment:
Minimizing/renaming unconditionally means that the type displayed in debugger is lying to use.

Reply:
Yes, but its only a white lie :) (And also would have to right? If we were ever to have variables with less than 1 gazillion ids?)

Reply:
Maybe so, but I foresee a frustrating hour this winter, sitting in my cold office late at night with the rain hammering against my window, trying to debug something with effects and being duped by this evil `toString`...

I propose you add control for this in `FormatOptions.scala`. It already has a parameter for controlling how variable names are handled.

Reply:
This apparently already works because Type calls `formatTypeWithOptions`. Is that okay or do you want to restructure?

Reply:
I'd still prefer to restructure. My thought is that the only job of `formatType` is to extract the options from the `flix` parameter.

Reply:
How about I let you restructure post merge?

Reply:
> How about I let you restructure post merge?

Ok